Cargo.toml

```diff
@@ -42,6 +43,7 @@ pin-project-lite = "0.2"
 tokio = { version = "1.24.2", optional = true, default-features = false }
 xz2 = { version = "0.1.6", optional = true }
 zstd-safe = { version = "6", optional = true, default-features = false }
+deflate64 = { version = "0.1.2", optional = true }
```

Comment:
This dependency looks quite new and its doc is a bit lacking.

I'm concern about if it will get enough maintanence.

Reply:
That's my crate and I'll maintenance as possible. I'm going to add more docs.

Reply:
Yes I noticed that, but I am still concerned about the bus factor.

I have two suggestions on this:
 - Invite people to your repository and its crates.io package to reduce bus factor
 - Optionally, setup automatic release if you don't want to invite them to the crates.io package
 - Give people you trusted with admin access

Alternatively, you can consider making this a rust-lang projects and that will make everybody on rust-lang able to maintain it.

Reply:
I have to agree with @NobodyXu here since this crate is widely used. I'm tracking the discussion on https://github.com/zip-rs/zip about deflate64 support; if they had it we'd probably prefer to use it.

I'd also accept this PR given co-ownership of `deflate64` on crates.io.

Reply:
@robjtede We should also do a review of `deflate64` and it will be great if we can have a few protests to assert that anything compressed by zip on windows can be decompressed by `deflate64` to original crates.

Unfortunately I'm not very familiar with deflate64 and other algorithms, so I would have a look at the code but I would definitely want someone who also know deflate64 to be the maintainer of `deflate64`.

Reply:
> We should also do a review of deflate64

> definitely want someone who also know deflate64 to be the maintainer

For sure; and I'm not in any rush to merge this so IMO lets see if zip-rs makes any movement in the next week.

Reply:
> Invite people to your repository and its crates.io package to reduce bus factor

Sorry, I don't have friends who write rust so I could not invite friends to my repository.

> Alternatively, you can consider making this a rust-lang projects and that will make everybody on rust-lang able to maintain it.

I don't know how to make an project a rust-lang project but If there is one way to do that, I think this is better since I have no friends. Could you let me know how to do this?

Reply:
> Sorry, I don't have friends who write rust so I could not invite friends to my repository.

You can invite @robjtede or me.

> I don't know how to make an project a rust-lang project but If there is one way to do that, I think this is better since I have no friends. Could you let me know how to do this?

I'm not sure either, but you can ask on zulip for this and you will get answers quickly.

Reply:
```bash
cargo owner --add <username>
# or
cargo owner --add github:<team>
```

Reply:
Thanks @anatawa12 I've accepted the invitation.